azavea/django-ecsmanage
# Initialize Django module for one-off management commands

## Overview

Turn the one-off task runner from https://github.com/open-apparel-registry/open-apparel-registry into its own Django module.

When reviewing this PR, take a look at @jeancochrane's comments [here](https://github.com/azavea/django-ecsmanage/pull/1/files).
 
Closes https://github.com/azavea/operations/issues/296.

## Testing instructions

### Run app tests

* Follow the instructions in the README to run tests locally: https://github.com/azavea/django-ecsmanage/blob/feature/jrb/init-package/README.md#developing

### Test the package against OAR

* Checkout this branch on OAR: https://github.com/open-apparel-registry/open-apparel-registry/tree/feature/jrb/test-ecsmanage-package
* Follow the setup instructions to create a dev environment
* Run `./scripts/manage ecsmanage --env staging help` and confirm that the task gets created as expected
